clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go

```diff
@@ -288,7 +288,7 @@ func GetReplicasFromRuntimeObject(obj runtime.Object) (int32, error) {
 		}
 		return 0, nil
 	case *appsv1.DaemonSet:
-		return 0, nil
+		return int32(numNodes), nil
```

Comment:
I don't think that's a good way of doing this. In DaemonSet you can set node-selector and then less pods can be created.

So instead of that, I think we should pass a ClientSet to this method and here list nodes with a given label-selector and return this number. I know this has a performance implication, but that isn't something that will be called all the time, and it will be more correct.

Reply:
Done, PTAL